test/e2e/testdata/acme/clusterissuer_gcp.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: acme-dns01-clouddns-ambient
```

Comment:
Please add a namespace field here, any namespace value should be good eg. `namespace: hello` / `namespace: foo` etc. to avoid the test panics in dynamic_resource_loader create resource call based upon what we discussed offline.

Reply:
You could also change current dynamic_resources.go:
https://github.com/openshift/cert-manager-operator/blob/3df88ce3d01c3e9ab8d919266c4ca8bef9dc1962/test/library/dynamic_resources.go#L81-L88

into:
```go
        if mapping.Scope.Name() == meta.RESTScopeNameNamespace {
                if overrideNamespace != "" {
			unstructuredObj.SetNamespace(overrideNamespace)
		}

		require.NotEmpty(d.t, unstructuredObj.GetNamespace(), "Namespace can not be empty!")

		
		dri = d.DynamicClient.Resource(mapping.Resource).Namespace(unstructuredObj.GetNamespace())
	} else {
```
i.e. to change the sequence, add the `if ...` before the `require.NotEmpty(...)`.